CHANGES.rst

```diff
@@ -1,3 +1,156 @@
+
+ixdat 0.2.9
+===========
+
+
+API changes
+-----------
+
+readers
+^^^^^^^
+- ``BiologicReader`` can now also read biologic .mpr files using an external package.
+  When reading a file ending in ".mpr", it first tries the ``galvani`` package, which
+  seems to work for LSV, CA, and CVA files. If that fails, it tries the ``eclabfiles``
+  package, which seems to work for OCV and CP files. See:
+  - https://github.com/echemdata/galvani
+  - https://github.com/vetschn/eclabfiles
+  These packages are not added as a requirement, but instead imported dynamically.
+  If the user tries to read a .mpr file without the needed package installed, they are
+  pointed to the package but also encouraged to export and read ".mpt" files instead.
+  ".mpr" files are recognized as biologic, and ``reader="biologic"`` works for both types.
+  Resolves `Issue #132 <https://github.com/ixdat/ixdat/issues/132`_
+  With `PR #134 <https://github.com/ixdat/ixdat/pull/134>`_
+
+
+- Zilien MS spectrum reader fixed.
+  Resolves `Issue #117 <https://github.com/ixdat/ixdat/issues/117>`_
+
+- ``Spectrum.read(reader="zilien")`` rather than ``reader="zilien_spec"`` as
+  before for reading in a zilien spectrum. This is accomplished by different
+  groups of reader for ``Spectrum.read`` and ``Measurement.read``
+  Also, zilien-read spectra now know their duration.
+
+- ``Measurement.read(..., reader="zilien")`` now returns a ``SpectroMSMeasurement``
+  when the reader can find zilien mass scans taken during the measurement. It
+  looks for the mass scans folder as zilien names it.
+  The default plotter is a ``SpectroMSPlotter`` which includes the MS spectra
+  data in a separate panel. The spectra are accessible by:
+
+    meas = Measurement.read("my_MID_with_spectra.tsv", reader="zilien")
+    meas.spectrum_series[0].plot()
+
+  which plots the first MS spectrum.
+  To leave out the mass scan data, include the argument ``include_spectra=False``
```

Comment:
I'm confused, I just tested it and had to actively include it (i.e. pass` include_spectra=True`) in order for it to read the spectra and return a `SpectroMSMeasurement`. 

Also, here you call it `SpectroMSMeasurement` but a few lines down `MSSpectroMeasurement` - which one is it now?

Reply:
It's now `MSSpectroMeasurement` , will fix!
I noticed that but thought I had fixed it so that it would automatically read spectra again! (It does now when there's also EC in the dataset.) Will fix.